inputs/supply/renewable_electricity/solar_power/capacity_of_households_solar_pv_solar_radiation.ad

```diff
@@ -0,0 +1,22 @@
+# Set the total output capacity of small-scale PV installations on household rooftops
+# Maximum value is set at twice the maximum amount of solar panels that can fit on the total roof surface area
+- query =
+    EACH(
+      UPDATE(
+        V(households_solar_pv_solar_radiation),
+        number_of_units,
+        USER_INPUT() / V(households_solar_pv_solar_radiation, electricity_output_capacity)
+      ),
+      UPDATE(
+        V(households_solar_pv_solar_radiation),
+        preset_demand,
+        V(households_solar_pv_solar_radiation, production_based_on_number_of_units)
+      )
+    )
+- priority = 0
+- max_value_gql = present:2*PRODUCT(DIVIDE(AREA(residences_roof_surface_available_for_pv),V(households_solar_pv_solar_radiation, land_use_per_unit)),V(households_solar_pv_solar_radiation, typical_electricity_production_per_unit))
```

Comment:
Note that with this new max_value I can now install 233.47 * 10^9 MW of household solar panels in a blank nl2019 scenario. With the previous slider settings I could at most install 37400.0 MW. I would suggest looking at the max_value of wind turbines and using something similar or change `typical_electricity_production_per_unit` to `electricity_output_capacity`.